### PR TITLE
init, log metrics on gptq

### PR DESCRIPTION
**Pull request by horheynm**
SUMMARY:
"please provide a brief summary"

TEST PLAN:
"please outline how the changes were tested"

### CODE REVIEW

File: src/llmcompressor/logger.py [hunk 1]
```diff
@@ -63,23 +63,22 @@ def configure_logger(config: Optional[LoggerConfig] = None):
     :param config: The configuration for the logger to use.
     :type config: LoggerConfig
     """
+    if (disabled := os.getenv("LLM_COMPRESSOR_LOG_DISABLED")) is not None:
```
Comment (bfineran) [inline src/llmcompressor/logger.py:66]: should make it possible to disable metric logs

File: src/llmcompressor/modifiers/quantization/gptq/utils/gptq_wrapper.py [hunk 2]
```diff
@@ -222,8 +227,22 @@ def compress(
             else:
                 W[:, i2:] -= w_err
 
-        logger.info("time %.2f" % (time.time() - tick))
-        logger.info("error %.2f" % torch.sum(Losses).item())
+        logger.log("METRIC", "time %.2f" % (time.time() - tick))
+        logger.log("METRIC", "error %.2f" % torch.sum(Losses).item())
+
+        gpu_usage = get_GPU_memory_usage()
```
Comment (bfineran) [inline src/llmcompressor/modifiers/quantization/gptq/utils/gptq_wrapper.py:233]: is it possible to lazy compute so we only run this when the metric logs are fired?

File: src/llmcompressor/utils/metric_logging.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            mem_info = pynvml.nvmlDeviceGetMemoryInfo(handle)
+            memory_usage_percentage = mem_info.used / mem_info.total
+            total_memory_mb = mem_info.total / (1024**2)
+            usage.append(
+                (memory_usage_percentage, total_memory_mb),
+            )
+        pynvml.nvmlShutdown()
+        return usage
+
+    except Exception:
```
Comment (bfineran) [inline src/llmcompressor/utils/metric_logging.py:27]: change to `except ImportError`

**Review by bfineran — APPROVED** [review]
